CONTRIBUTING.md

```diff
@@ -153,6 +153,17 @@ At this point the pre-commit checks will run against any files that
 you attempt to commit.  If you want to run the checks against the
 entire repo, just execute `pre-commit run --all-files`.
 
+## Quality Assurance/Code reviews ##
+
+In order to maintain standardization with practices, ensure security standards
+are being met, and to incorporate third party code as seamlessly as possible,
+submitted code will go through our quality assurance team.
+Code contributors are welcome to start communications with QA as early as
+project inception, but will need to engage prior to any code submissions.
+
+Third parties are responsible for coordinating initial code reviews between
+members of their support teams to ensure quality code is merged
```

Comment:
Also, I think there is some conflation between "code contributors" and "third parties" here.  We need to either use the same nomenclature everywhere or make the definitions clear.

Reply:
This conversation still hasn't been addressed.  First the language needs to be made clearer by @jmorrowomni, since I don't fully understand what is being said, and after that @cisagov/team-ois needs to acknowledge that it agrees with the content.

Reply:
@jmorrowomni - have you had a chance to clarify the wording here?  Do you have any questions or blockers that we can assist you with?